DBD-Exam-Project/lib/DTO/PrescriptionDto.cs

```diff
@@ -14,8 +14,11 @@ public class PrescriptionDto
         public string? MedicineName { get; set; }
         public PersonDto? Patient { get; set; }
         public PersonDto? Doctor { get; set; }
+        public int? PatientId { get; set; }
+        public int? DoctorId { get; set; }
```

Comment:
De her er under `Doctor` og `Patient` properties, der er ikke nogen grund til at have dem to gange